index.html

```diff
@@ -24,19 +27,159 @@ <h5>CB</h5>
+            don’t hestiate to contact me.
         </p>
-        <h4>LET'S CONNECT</h4> 
+        <h4 class="connect">LET'S CONNECT</h4>
         <ul class="socials">
-            <li><a href="https://github.com/Christianib003"><img class="social" src="./Images/GutHub.png" alt="github"></a></li>
-            <li><a href="https://www.linkedin.com/in/christian-iradukunda-byiringiro-657598226"><img class="social resize"
-                    src="./Images/png-transparent-computer-icons-social-media-linkedin-slideshare-linkedin-gray-angle-text-rectangle-thumbnail.png"
-                    alt="linkedIn"></a></li>
-            <li><a href="#"><img class="social" src="./Images/M.png" alt="M"></a></li>
-            <li><a href="#"><img class="social" src="./Images/Peace.png" alt="peace"></a></li>
+            <li><a href="https://github.com/Christianib003"><i class="fab fa-github"></i></a></li>
+            <li><a href="https://www.linkedin.com/in/christian-iradukunda-byiringiro-657598226"><i
+                        class="fab fa-linkedin-in"></i></a></li>
+            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fab fa-instagram"></i></a></li>
+        </ul>
+    </section>
+    <!-- second page -->
+    <section class="Works">
+        <!-- card 1 -->
+        <div class="card">
+            <img class="image" src="./Images/Snapshoot-page-2.png" alt="tonic">
+            <h2>Tonic</h2>
+            <ul class="triple">
+                <li class="canopy">CANOPY</li>
+                <li class="double">Back End Dev</li>
+                <li class="double">2015</li>
+            </ul>
+            <p class="paragraph">
+                A daily selection of privately personalised reads; no accounts or no sign-ups required.
+            </p>
+            <div class="langs">
+                <div class="lang1 languages">html</div>
+                <div class="lang2 languages">css</div>
+                <div class="lang3 languages">javascript</div>
+            </div>
+            <button type="button" class="seeProject">See Project</button>
+        </div>
+        <!-- card 2 -->
+        <div class="card">
+            <img class="image" src="./Images/Snapshoot-page-3.png" alt="tonic">
+            <h2>Multi-Post Stories</h2>
+            <ul class="triple">
+                <li class="canopy">CANOPY</li>
+                <li class="double">Back End Dev</li>
+                <li class="double">2015</li>
+            </ul>
+            <p class="paragraph">
+                A daily selection of privately personalised reads; no accounts or no sign-ups required.
+            </p>
+            <div class="langs">
+                <div class="lang1 languages">html</div>
+                <div class="lang2 languages">css</div>
+                <div class="lang3 languages">javascript</div>
+            </div>
```

Comment:
- [ ] As pointed by the previous reviewer you need here to use `<ul>` tag instead of a `<div>` tag, this is one of the main requirements and that will make your code more semantic. Kindly work on refactoring that🙏🏼